prow/cmd/deck/tide.go

```diff
@@ -57,16 +57,11 @@ type tideAgent struct {
 }
 
 func (ta *tideAgent) start() {
-	startTimePool := time.Now()
-	if err := ta.updatePools(); err != nil {
-		ta.log.WithError(err).Error("Updating pools the first time.")
-	}
-	startTimeHistory := time.Now()
-	if err := ta.updateHistory(); err != nil {
-		ta.log.WithError(err).Error("Updating history the first time.")
-	}
-
 	go func() {
+		startTimePool := time.Now()
+		if err := ta.updatePools(); err != nil {
+			ta.log.WithError(err).Error("Updating pools the first time.")
+		}
```

Comment:
Instead of duplicating these sections just move the `time.Sleep` to the end of the loop so that we update before sleeping.